### PR TITLE
Adds the Stechkin APS to the nukeop uplink

### PR DESCRIPTION
**Pull request by Bm0n**
## What Does This PR Do
Adds the already existing but unused Stechkin APS to the nukeop uplink for 8 TC

Tweaks the Stechkin APS in numerous ways including:

Changing its caliber from 9mm to 10mm effectively buffing its damage.

Upping its magazine count from 15 to 20 rounds.

Gives the APS specialty magazine options identical to the Stechkin pistol(HP, AP, FIRE).

## Why It's Good For The Game

The APS fills a gap in the nukeops arsenal acting as a backup weapon a cut above the regular Stechkin pistol. It can be incredibly deadly in the right hands, especially when utilized dual wielded.

I had this PR in mind for a while now as the APS is something that TG added to their uplink... but I was surprised to find out that TG actually removed the APS a few weeks ago citing that it was "superfluous". Personally, I find that ridiculous as I have seen it used plenty of times on TG to significant effect.

Just in case though I decided to change up the implementation, lowering the price from TG by 2 TC to 8 and upping the magazine count from 15 to 20 rounds per mag. I think this makes the gun an attractive option now while keeping it well balanced.
## Images of changes

Specialty mag sprites added for the APS, fire as an example.
image

## Testing

I ran it and tested it in the game. Should be good.
## Changelog
:cl: Bmon
add: The Stechkin APS has been added to the nukeop uplink for 8 TC!
/:cl:

### CODE REVIEW

**Comment by Bm0n** [comment]
I don't know if I got the TC price per mag right, they're all 1 TC higher than the pistol equivalent which makes sense but compared to other mags is pretty pricey, especially for the specialty ones. I'd like to get some opinions from the balance team about that.

**Comment by Sirryan2002** [comment]
Waiting to do final review until author or design team are confident on the chosen TC values.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -448,6 +448,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
 	surplus = 50
 
+/datum/uplink_item/dangerous/aps
+	name = "Stechkin APS Pistol"
+	reference = "APS"
+	desc = "The automatic machine pistol version of the FK-69 'Stechkin' chambered in 10mm Auto with a detachable 20-round box magazine. Perfect for dual wielding or as backup."
+	item = /obj/item/gun/projectile/automatic/pistol/APS
+	cost = 8
+	gamemodes = list(/datum/game_mode/nuclear)
+	surplus = 40
```
Comment (hal9000PR) [inline code/datums/uplink_item.dm:458]:
```suggestion
/datum/uplink_item/dangerous/aps
	name = "Stechkin APS Pistol"
	reference = "APS"
	desc = "The automatic machine pistol version of the FK-69 'Stechkin' chambered in 10mm Auto with a detachable 20-round box magazine. Perfect for dual wielding or as backup."
	item = /obj/item/gun/projectile/automatic/pistol/APS
	cost = 8
	gamemodes = list(/datum/game_mode/nuclear)
```
nukies cant get surplus creates.

**Review by hal9000PR — CHANGES_REQUESTED** [review]

File: code/datums/uplink_item.dm (hunk 1 above)
Comment (Bm0n) [reply code/datums/uplink_item.dm:458]: huh, I thought nukeop gear could spawn in surplus crates. Either way, I removed it.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**